src/amo/components/UserProfileEditPicture/styles.scss

```diff
@@ -46,7 +46,16 @@ $picture-size: 128px;
   }
 
   .UserProfileEditPicture-delete-button {
-    cursor: pointer;
-    width: $picture-size;
+    display: none;
```

Comment:
i know you made note about this but was wondering if we should keep this. I guess i thought it was always better to have the same functionality - as much as possible - between the different breakpoints, no?

Reply:
maybe yeah

Reply:
even on the same device it could show up on landscape orientation and not portrait so just wanted to make sure it's what we want : )

Reply:
Ok, that was silly. Thanks for catching this, I will make the button always available.